disnake/ext/commands/common_bot_base.py

```diff
@@ -463,7 +463,7 @@ def _resolve_name(self, name: str, package: Optional[str]) -> str:
         try:
             return importlib.util.resolve_name(name, package)
         except ImportError:
-            raise errors.ExtensionNotFound(name)
+            raise errors.ExtensionNotFound(name) from None
```

Comment:
this should keep the original error

Reply:
https://github.com/DisnakeDev/disnake/pull/950/commits/a31297b5838c1746d1d663b4a7c907eb2437d7f1